circuit-breaker/src/main/java/com/iluwatar/circuitbreaker/App.java

```diff
@@ -63,22 +64,51 @@
    *
    * @param args command line args
    */
-  @SuppressWarnings("squid:S2189")
   public static void main(String[] args) {
-    //Create an object of monitoring service which makes both local and remote calls
-    var obj = new MonitoringService();
-    //Set the circuit Breaker parameters
-    var circuitBreaker = new CircuitBreaker(3000, 1, 2000 * 1000 * 1000);
+
     var serverStartTime = System.nanoTime();
-    while (true) {
-      LOGGER.info(obj.localResourceResponse());
-      LOGGER.info(obj.remoteResourceResponse(circuitBreaker, serverStartTime));
-      LOGGER.info(circuitBreaker.getState());
-      try {
-        Thread.sleep(5 * 1000);
-      } catch (InterruptedException e) {
-        LOGGER.error(e.getMessage());
-      }
+
+    var delayedService = new DelayedRemoteService(serverStartTime, 5);
+    //Set the circuit Breaker parameters
```

Comment:
I think these kind of comments don't add any value and should be removed

Reply:
Agreed. removed.